webapp/src/client/app/modules/web-client/vnc/web-client-vnc.component.ts

```diff
@@ -412,12 +412,12 @@ export class WebClientVncComponent extends WebClientBaseComponent implements OnI
       configBuilder.withDesktopSize(connectionParameters.screenSize);
     }
 
-    if (connectionParameters.disableCursor) {
-      configBuilder.withExtension(disableCursor(true));
+    if (connectionParameters.enableCursor) {
+      configBuilder.withExtension(enableCursor(true));
     }
 
-    if (connectionParameters.disableExtendedClipboard) {
-      configBuilder.withExtension(disableExtendedClipboard(true));
+    if (connectionParameters.enableExtendedClipboard) {
+      configBuilder.withExtension(enableExtendedClipboard(true));
```

Comment:
nitpick: it seems reasonable to remove the `if`:
```ts
configBuilder.withExtension(enableExtendedClipboard(connectionParameters.enableExtendedClipboard));
```
And this can go in the call chain above